src/lib/utils/date_time_utils.hpp

```diff
@@ -8,19 +8,14 @@
 namespace opossum {
 
 /**
- * Takes delimited date string with order year-month-day (ISO 8601 extended format), e.g., "2001-01-01".
+ * Takes delimited timestampt string with order year-month-day [hour:minute:second<.second fraction>] (ISO 8601 extended
+ * format), e.g., "2001-01-01", "2000-01-01 13:01:02" or "2000-01-01 13:01:02.5000". Boost allows out-of-bounds values
+ * for timestamps, e.g., '2000-01-01 25:61:61' is a valid date time and the overflow is added to the subsequent time
```

Comment:
Do we want to differentiate here between timestamp and data time? How about `is a valid timestamp`? or `is valid`?